ZupIT/ritchie-cli
# Improvement pipeline

- What I did
I changed pipeline so we can now run stuff on qa, **staging** and production using new AWS account organization proposed by **Zup**

- How to verify it
See if the pipeline runs smoothly on staging and properly on the other environments.
I already tested the new pipeline and it worked according to plan for **PRs** and for **delivery branches**, i don't know if I'm missing any scenarios, probably not.

- Description for the changelog

Adding pipeline, simplifying routing scripts and removing old stuff that we no longer use.
